src/pages/resultsView/network/files/mutationMapper.js

```diff
@@ -1115,12 +1115,18 @@ var MutationDetailsTableFormatter = (function()
 		var style = "";
 		var tipStyle = "";
 
-		if (frequency)
+		if (frequency > 0.01)
 		{
 			style = "mutation_table_allele_freq";
 			text = frequency.toFixed(2);
 			tipStyle = tipClass;
 		}
+		else if (frequency < 0.01) 
```

Comment:
@coderrsid Thanks for taking care of this issue! We may don't want to miss out the 0.01 case. If we use `else if (frequency < 0.01)` then 0.01 will not have text. You may want to use `else`.

Reply:
Done @dippindots